### DISCUSSION TITLE
missing mx.postal.example.com A record in dns configuration?

### DISCUSSION THREAD

**Discussion by kineticz** [postalserver/postal#1623] [Help with using Postal]
I struggled with the incoming messages for a few hours and found out that i need a A record for mx.postal.example.com, after which I could receive emails. Not 100% sure if this was rather fixed by some other changes I made (tried too many of them and don't remember exactly any more). If the A record is needed indeed, then it's right now missing in the documentation. 

In my tests, without the A record for mx.postal.example.com, sending an email to something@example.com would give this error: 

> No SMTP servers were available for mail-coskenn.com. Tried mx.mail.coskenn.com
> Could not resolve mx.mail.coskenn.com
> Support Ref: ONBZIF1V

**Comment by willpower232** [comment]
Yes, if a domain has mx.postal.example.com in its MX record then you need an A record for mx.postal.example.com to point it at an IP address (replace example.com with the domain of your Postal server)
